domains/admin-portal/AdminPortal.Web/AdminPortal.Tests/Controllers/WhitelistedOrganizationsControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using AdminPortal._Features_;
+using AdminPortal.Dtos.Response;
+using AdminPortal.Tests.Setup;
+using Microsoft.AspNetCore.Mvc.Testing;
+using NSubstitute;
+
+namespace AdminPortal.Tests.Controllers;
+
+public class WhitelistedOrganizationsControllerTests
+{
+    [Fact]
+    public async Task Given_WhitelistedOrganizations_When_GetIndex_Then_ReturnsSuccessfulResponse()
+    {
+        var factory = new TestWebApplicationFactory();
+        var client = factory.CreateAuthenticatedClient<GeneralUser>(new WebApplicationFactoryClientOptions(), 12345);
+
+        var response = await client.GetAsync("/WhitelistedOrganizations", TestContext.Current.CancellationToken);
+
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+    }
+
+    [Fact]
+    public async Task Given_ValidTin_When_PostingToWhitelistEndpoint_Then_ReturnsRedirect()
```

Comment:
Will this test fail if we change the location header in authorization sub system? If so, then i dont think we should have this test